source/_patterns/01-molecules/teasers/teaser.yaml

```diff
@@ -89,13 +89,14 @@ schema:
       properties:
         meta:
           $ref: ../../01-molecules/components/meta.yaml#/schema
-        publishState:
+        formats:
           type: object
+          minProperties: 1
           properties:
-            vor:
+            html:
```

Comment:
with respect to an array `['html', 'pdf']`, makes clear that the input order doesn't count as it's decided in the template